packages/SystemUI/res/values/cm_strings.xml

```diff
@@ -46,6 +46,9 @@
     <string name="global_screenshot_selector_capture_description">Capture partial screenshot</string>
     <string name="global_screenshot_selector_cancel_description">Cancel partial screenshot</string>
 
+    <!-- Power menu QS tile -->
+    <string name="quick_settings_power_menu_label">Power menu</string>
```

Comment:
Move to cr_strings to be able to translate

Reply:
Done